terraform/aks/databases.tf

```diff
@@ -0,0 +1,18 @@
+module "postgres" {
+  source = "git::https://github.com/DFE-Digital/terraform-modules.git//aks/postgres?ref=testing"
```

Comment:
Can we use terrafile so we can use different module branches per environment?
See:
https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/terraform/application/database.tf#L2
https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/terraform/application/config/development_Terrafile
https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/Makefile#L30-L32
https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/Makefile#L40